Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -82,6 +82,16 @@ class Digitizer
   double mLastTime = 1.0e10; // starts in the future
   int mEventID = 0;
   int mSrcID = 0;
+  int kNpad = 0;            // Number of pads included in the pad response
+  float kAmWidth = 0;       // Width of the amplification region
+  float kDrWidth = 0;       // Width of the drift retion
+  float kDrMin = 0;         // Drift + Amplification region
+  float kDrMax = 0;         // Drift + Amplification region
+  int timeBinTRFend = 0;    // time bin TRF ends
+  int maxTimeBins = 30;     // Maximum number of time bins for processing signals, usually set at 30 tb = 3 microseconds
+  int maxTimeBinsTRAP = 30; // Maximum number of time bins for processing adcs; should be read from the CCDB or the TRAP config
+  float samplingRate = 0;
+  float elAttachProp = 0;
```

Comment:
these variables don't follow the naming rule for class members. And in case some of them are actual constants, please make them constexpr static.

Reply:
Sorry, I forgot to follow the naming conventions at this point. I added a couple additional changes in the same direction, too. Nice that we have o2checkcode.